src/lib/libwasi.js

```diff
@@ -362,7 +362,7 @@ var WasiLibrary = {
   fd_seek__i53abi: true,
   fd_seek: (fd, offset, whence, newOffset) => {
 #if SYSCALLS_REQUIRE_FILESYSTEM
-    if (isNaN(offset)) return {{{ cDefs.EOVERFLOW }}};
+    if (isNaN(offset)) return -{{{ cDefs.EOVERFLOW }}};
```

Comment:
Wait, don't musl syscalls need `-` but not wasi ones? This PR adds the first `return -{{{` into this file.

Reply:
Oops, you are correct.  Reverted this file.